image/templates/helm/shared/templates/99-shared-openshift-monitoring.yaml

```diff
@@ -0,0 +1,80 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.monitoring.openshift.enabled -}}
+  {{- if not ._rox.monitoring.openshift._securedWithCentral -}}
+---
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "srox.globalResourceName" (list . "stackrox-cluster-ext-api-auth-reader") }}
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+annotations:
+  openshift.io/reconcile-protect: false
+rules:
+- apiGroups:
+    - ""
+  resources:
+    - configmaps
+  resourceNames:
+    - extension-apiserver-authentication
+  verbs:
+    - get
+    - list
+    - watch
+  {{- end -}}
+  {{- if (or ._rox.scannerV4._indexerEnabled ._rox.scannerV4._matcherEnabled) -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "rhacs-scanner-v4-auth-reader-{{ .Release.Namespace }}"
```

Comment:
Shall we use the `srox.globalResourceName` here as well? Then `ClusterRoleBinding` and `ClusterRole` are handled consistently.

Reply:
Potentially. But, the bindings were already prefixed with `rhacs-` for Central and Sensor, so I decided to keep them the same and use the same name prefix for Scanner V4 to keep it consistent. (The `globalResourceName` requires resources to be prefixed `stackrox-`.)